projects/docs/docs/guides/third-party-dependencies.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: External dependencies
-slug: '/guides/third-party-dependencies'
+slug: '/guides/external-dependencies'
```

Comment:
From the eyes of a search engine crawler, this is a breaking change and we might get a bad rating for changing URLs without setting up permanent redirects. Therefore I'd either keep the previous slug or set up a redirect.

Reply:
reverted